plugins/modules/zabbix_maintenance.py

```diff
@@ -301,14 +311,24 @@ def main():
     else:
         zabbix_host = "host"
 
+    if not name:
+        name = "Created at " + now.strftime('%Y-%m-%d %H:%M')
+
     changed = False
 
     if state == "present":
         if not host_names and not host_groups:
             module.fail_json(msg="At least one host_name or host_group must be defined for each created maintenance.")
 
-        now = datetime.datetime.now().replace(second=0)
-        start_time = time.mktime(now.timetuple())
+        if start_time:
+            try:
+                time_obj = datetime.datetime.strptime(start_time, '%Y-%m-%d %H:%M')
+            except (Exception, ValueError) as e:
```

Comment:
```suggestion
            except (Exception, ValueError):
```

unused `e` is troubling tox linters